root/usr/local/sbin/occ

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2020 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+# OCC wrapper running PHP from SCL
+
+sudo -u apache scl enable rh-php73 -- php -d memory_limit=512M  /usr/share/nextcloud/occ "$@"
```

Comment:
As `occ` is under `sbin/` and I guess it already runs as root. The  `runas` command provides a more lightweight alternative than `sudo` to drop privileges.

Reply:
I don't get it.
What is the proposal? :thinking:

Reply:
Something like

    runuser -s /bin/bash -c "/usr/bin/scl enable rh-php56 -- php /usr/share/tancredi/scripts/upgrade.php" - apache

Reply:
IMO current implementation is cleaner.
Also I didn't find a clean way to pass all escaped commands using `runuser` (I've just spent a couple of minutes on it).
If there is no real advantage, I will leave it unchanged.

Still this is not the important part of this PR :D